I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -2886,7 +2941,7 @@ module: ietf-optical-impairment-topology
        |        +--ro otsi-ref* []
        |        |  +--ro otsi-carrier-ref?   leafref
        |        |  +--ro e2e-mc-path-ref*    leafref
-       |        +--ro delta-power?      l0-types:power-in-dbm-or-null
+       |        +--ro delta-power?      l0-types:gain-in-db-or-null
```

Comment:
delta-power contains a ratio in dB. one may call it gain, however gain is only positive, where delta_p could be negative. so the type should be changed